src/app.scss

```diff
@@ -18,10 +18,21 @@
     left: 50%;
     transform: translate(-50%, -50%);
     width: 600px;
-    height: 500px;
+    height: 600px;
     display: flex;
     flex-direction: vertical;
     justify-content: center;
     align-items: center;
+
+    &__buttons {
+      position: absolute;
```

Comment:
is position absolute needed? the parent flex should take care of the positioning